16/06/2022 17:38:59 The log file has been set-up.
16/06/2022 17:38:59 Worker process running with:
16/06/2022 17:38:59  - Job Name: process
16/06/2022 17:38:59  - PID: 1601
16/06/2022 17:38:59  - Threads: 1
16/06/2022 17:38:59  - Node: rosetta6
16/06/2022 17:38:59  - System: Linux
16/06/2022 17:38:59  - Machine: x86_64
16/06/2022 17:38:59  - Timeout Limit: 356400
16/06/2022 17:38:59  - Process: [memmap(['/Mounts/rbg-storage1/users/ayed/shapepipe_data/inputs/real_SNR/mid_density/L256/shapepipe_run_2022-06-16_16-24-50/merge_starcat_runner/output/full_starcat-0000000.fits'],
       dtype='<U167')]
16/06/2022 17:38:59  - Running module: mccd_plots_runner
16/06/2022 17:38:59 TOTAL e1 residual RMSE: 8.376506e-03

16/06/2022 17:38:59 TOTAL e2 residual RMSE: 7.289206e-03

16/06/2022 17:38:59 TOTAL R2 residual RMSE: 1.983962e-01

16/06/2022 17:39:05 Bins: e1 residual RMSE: 0.004166

16/06/2022 17:39:15 Bins: e2 residual RMSE: 0.004336

16/06/2022 17:39:25 Bins: R2 residual RMSE: 0.015769

16/06/2022 17:39:35 [('ra', 'dec', 'g1', 'g2', 'psf_g1', 'psf_g2', 'w')]
16/06/2022 17:39:35 [('ra', 'dec', 'g1', 'g2', 'psf_g1', 'psf_g2', 'w')]
16/06/2022 17:39:35 [('ra', 'dec', 'sigma', 'psf_g1', 'psf_g2', 'psf_sigma', 'w')]
16/06/2022 17:39:35 [('ra', 'dec', 'g1', 'g2', 'sigma', 'psf_g1', 'psf_g2', 'psf_sigma', 'w')]
16/06/2022 17:39:35 [('ra', 'dec', 'sigma', 'psf_g1', 'psf_g2', 'psf_sigma', 'w')]
16/06/2022 17:39:35  > Computing rho statistics...
16/06/2022 17:39:35 ERROR: Cannot use given fields: ["{'coords':", "'spherical',", "'metric':", "'Euclidean',", "'sep_units':", "'arcmin',", "'bin_type':", "'Log'}"]
Traceback (most recent call last):
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/worker_handler.py", line 180, in _run_worker
    with_timeout(self.worker_dict['timeout'], self.w_log.name)(
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/timeout.py", line 42, in inner
    return decorated(*args, **kwargs)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/worker_handler.py", line 194, in _worker_execution
    self._run_module()
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/worker_handler.py", line 215, in _run_module
    self._stdout, self._stderr = self._module_runner(
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/modules/mccd_plots_runner.py", line 140, in mccd_plots_runner
    mccd_plots.rho_stats(
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/modules/mccd_package/mccd_plot_utilities.py", line 1508, in rho_stats
    rho_results = [
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/modules/mccd_package/mccd_plot_utilities.py", line 1509, in <listcomp>
    rho_stat(stilecat, config=TreeCorrConfig)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/sys_tests.py", line 819, in __call__
    return self.getCF('gg', new_data, new_data2, new_random, new_random2,
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/sys_tests.py", line 543, in getCF
    results = stile.ReadTreeCorrResultsFile(output_file)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/treecorr_utils.py", line 105, in ReadTreeCorrResultsFile
    return stile_utils.FormatArray(output, fields=fields)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/stile_utils.py", line 90, in FormatArray
    raise RuntimeError('Cannot use given fields: '+str(fields))
RuntimeError: Cannot use given fields: ["{'coords':", "'spherical',", "'metric':", "'Euclidean',", "'sep_units':", "'arcmin',", "'bin_type':", "'Log'}"]
16/06/2022 17:39:35 The log file has been closed.
